src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(num) {
+      this.result += num;
+    },
+
+    subtract(num) {
+      this.result -= num;
+    },
+
+    multiply(num) {
+      this.result *= num;
+    },
+
+    divide(num) {
+      if (num !== 0) {
+        this.result /= num;
+      }
```

Comment:
Consider throwing an Error or logging a warning when trying to divide by zero. Right now, the divide method silently fails, which might not be the expected behavior for the user of the calculator. For example: `if (num === 0) throw new Error('Cannot divide by zero.');`